Makefile

```diff
@@ -11,16 +11,24 @@ test:
 	go test ./...
 .PHONY: test
 
+integration-test:
+	./hack/integration-test.sh
+.PHONY: integration-test
+
+format:
+	go fmt ./...
+.PHONY: format
+
 clean:
 	rm -rf _output/
 	rm -rf bin
 .PHONY: clean
 
-update-codegen-crds:
-	go run ./vendor/github.com/openshift/library-go/cmd/crd-schema-gen/main.go --domain openshift.io --apis-dir vendor/github.com/openshift/api --manifests-dir install/
-update-codegen: update-codegen-crds
-verify-codegen-crds:
-	go run ./vendor/github.com/openshift/library-go/cmd/crd-schema-gen/main.go --domain openshift.io --apis-dir vendor/github.com/openshift/api --manifests-dir install/ --verify-only
-verify-codegen: verify-codegen-crds
+update-codegen:
+	./hack/update-codegen.sh
+
+verify-codegen:
+	./hack/verify-codegen.sh
+
 verify: verify-codegen
-.PHONY: update-codegen-crds update-codegen verify-codegen-crds verify-codegen verify
+.PHONY: update-codegen verify-codegen verify
```

Comment:
nit: dup `verify-codegen` entries?  Also, the pattern in the rest of the `Makefile` appears to be a new `.PHONY` entry after each target.  I'm agnostic about whether we use that or a single large `.PHONY` entry, but I'd rather not split between both patterns.

Reply:
I've just removed the codegen stuff entirely: the `update-codegen` was just a tombstone and we do not use it in our CI jobs: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-version-operator/openshift-cluster-version-operator-master.yaml